### ISSUE TITLE
Corridor/Region Shapes keep running for hours without any results

### ISSUE THREAD

**Issue by Mohieb-Nassar** [Autodesk/civilconnection#67]
I am trying this awesome package at different versions of Revit and i am facing the same problem.
The script keep running for hours without getting the Shapes .
I am wondering if anyone face the same issue. 
My Laptop has 64GB Rams and i7 Core VPRO CPU which is good enough for heavy files.
My test file is a very simple corridor  and the file size is very small.

![image](https://github.com/Autodesk/civilconnection/assets/76867171/b51d46e0-b124-419b-9ba2-2dc6a0eee833)

**Comment by argomo** [comment]
Exact same here!

**Comment by jrpindave** [comment]
Same, did you find the solution?

**Comment by paoloemilioserra** [comment]
have you installed and verified that CivilPython is working? This is an extra component that is in the release file and is required to extract the coordinates of the shapes.

**Comment by jrpindave** [comment]
> have you installed and verified that CivilPython is working? This is an extra component that is in the release file and is required to extract the coordinates of the shapes.

Yes! this did solve it!, i had the python bundle installed in C://Program Files (x86), i changed it to the normal Program Files and it worked, many thanks

**Comment by Mohieb-Nassar** [comment]
When i call the PYTHON Command in Civil3D i got "Unknown Command " and i still cant get any shapes from corridor or sub regions.  I have installed the python in the "C:\ProgramData\Autodesk\ApplicationPlugins" and in the "C:\Program Files\Autodesk\ApplicationPlugins" and getting the same issue. I can get the Civil3d file and read values but i cant get the shapes as in the snap below the dynamo keep loading without result.

![Image](https://github.com/user-attachments/assets/a645a5e7-1b03-448b-a1f1-8e634bd40b2a)

![Image](https://github.com/user-attachments/assets/3dde430c-38a5-4685-8201-d96cb4ce593f)

![Image](https://github.com/user-attachments/assets/00bf08b9-e0ea-496c-824b-4da5b13b12e5)

**Comment by paoloemilioserra** [comment]
you need to get the Python command working first. Have you read the guide about it and followed the procedure?
